build.sbt

```diff
@@ -17,7 +17,7 @@ lazy val scoverageSettings = {
 lazy val wartRemoverSettings = {
   val wartRemoverWarning = {
     val warningWarts = Seq(
-      Wart.JavaSerializable,
+      //Wart.JavaSerializable,
```

Comment:
What was the change that made disabling this check necessary?

Reply:
Enumeration json serialiser